cmake/3rdParty/FetchRGA.cmake

```diff
@@ -0,0 +1,26 @@
+# This script encapsulates fetching RGA as a dependency and providing the
+# requested targets. To use it, simply include this script from the CMakeLists.txt
+# file of the target platform.
+
+message(STATUS "started FetchRGA")
+
+if (NOT FETCHED_RGA)
+    include(FetchContent)
+    if (WIN32)
+        FetchContent_Declare(
+            RGA
+            URL https://github.com/GPUOpen-Tools/radeon_gpu_analyzer/releases/download/2.6.2/rga-windows-x64-2.6.2.zip
```

Comment:
Does this work with an installer? By that I mean if you produce an installer for O3DE by building INSTALL target, can you use this feature still? I am guessing this is a binary payload, does it exist in an installer SDK layout?

Reply:
I try to build CMakePredefinedTargets/INSTALL
But FetchGRA.cmake is not invoked

I include FetchGRA.cmake in 3rdParty.cmake, which is included in o3de/CMakeList.txt
And it is only included `if(NOT INSTALLED_ENGINE)`, following the logic of other 3rdParty cmake files
Should it be working with the installer? What is a better way to do this?

Reply:
FetchRGA.cmake will only be invoked when create/recreate the project.
It is included in 3rdParty.cmake, which is included in o3de/CMakeLists.txt.
I added in `if(NOT INSTALLED_ENGINE)`, to follow the manner of other 3rdParty cmake files

I try to build CMakePredefinedTargets/INSTALL
The downloaded binary payload will not be copied to the install folder, if that's what you ask

Reply:
If a binary wont be going to the installer folder then it means only users who are building from source code directly can use the tool. For example, if this goes in then the next O3DE installer wont have this functioning out of the box, right?

I believe there are are ways to add these binaries to install payloads as well. I would ask in sig-core. Depending on the sig that owns this PR, perhaps create a separate GHI for that work. 

Thanks!

Reply:
I think you can add this instruction to fix the installer. (Or something like that.)
```
ly_install_directory(DIRECTORIES .)
```

Reply:
Since this is mainly for graphics engineers to optimize shaders, it's probably okay for now to have it only available when building from source code. But I agree we should create a GHI to follow up by making this available to installed builds. Also, see my comment below about not failing the job when rga is missing.